.github/ISSUE_TEMPLATE/NEW_PROPOSAL.md

```diff
@@ -0,0 +1,50 @@
+
+---
+name: "\U0001F4D6 New Proposal"
+about: Template for creating a new proposal
+title:
+labels: 
+assignees: ''
+
+---
+<!--
```

Comment:
Are those comments going to be visible in the issue template? My understanding is that they will not, and we would like them to be visible.

Reply:
These comments won't be visible, but we can put the full template in a .md file as instructions?

Reply:
I think this would be good in the README.md file